core/controllers/library.py

```diff
@@ -333,13 +414,24 @@ class CollectionSummariesHandler(base.BaseHandler):
     """Returns collection summaries corresponding to collection ids."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'stringified_collection_ids': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            }
+        }
```

Comment:
Can we refactor the code to actually send proper list of strings instead of just string of separated values?

Reply:
SVS architecture depend entirely in schema_utils for normalizing the argumnets.
For handling the case as mentioned by you we have to update schema_utils which is not so much efficient approach.
Looking forward for your point.

Reply:
Sorry, not fully sure what you mean here.

Reply:
I mean for normalization of any data through Schema Validation System (SVS) we depend on schema_utils file.
So if we want to normalize ```stringified_collection_ids (string)``` into list of strings we must have to modify schema_utils.
But in schema_utiils we only want to keep generic normailizers not for any specific arguments ([as discussion here](https://github.com/oppia/oppia/pull/13370/files/d8018706b1b8d4f01c2de7dcc56e714191b68108#diff-3a5d5cbd63bf7a719c023d1c78bdfdbbbfd416e22716d96d874fd613326fa7f4))
Please help me if we can introduce this normalization in another way.

Reply:
@vojtechjelinek I think you cannot have the list or boolean param for GET or DELETE requests and that's why we have it this way! (not sure!)

Reply:
Hmm, should we use different type of requests here?

Reply:
I personally think it's fine to keep as str. You'd need to encode these complex structures somehow anyway for GET/DELETE (json or something) and comma-separated encoding seems reasonable (and I think somewhat standard?) in this case.

Reply:
Hmm, okay, I think it is fine to use this format then, but could we add a better validation and normalization for it?

Reply:
Added.